software/snpeff/main.nf

```diff
@@ -0,0 +1,57 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+
+params.options = [:]
+options = initOptions(params.options)
+params.use_cache = false
```

Comment:
Same comment as for [VEP](https://github.com/nf-core/modules/pull/547#discussion_r664373643)